--- /home/soramichi/anaconda3/lib/python3.5/site-packages/sklearn/ensemble/forest.py	2016-12-02 18:41:47.034231087 +0900
+++ forest.py	2016-12-02 18:41:14.974229419 +0900
@@ -182,7 +182,7 @@
 
         return np.array(results).T
 
-    def fit(self, X, y, sample_weight=None):
+    def fit(self, X, y, sample_weight=None, force_no_check=False):
         """Build a forest of trees from the training set (X, y).
 
         Parameters
@@ -209,7 +209,8 @@
             Returns self.
         """
         # Validate or convert input data
-        X = check_array(X, dtype=DTYPE, accept_sparse="csc")
+        if not force_no_check:
+            X = check_array(X, dtype=DTYPE, accept_sparse="csc")
         if issparse(X):
             # Pre-sort indices to avoid that each individual tree of the
             # ensemble sorts the indices.
